### ISSUE TITLE
Client side JavaScript version

### ISSUE THREAD

**Issue by ymerkos** [KoljaB/RealtimeSTT#145]
B"H
Fantastic library

Is there any way to get this working fast on the browser using wasm or something?

**Comment by KoljaB** [comment]
Not really. There are some implementations for whisper running fully in the browser. Yet RealtimeSTT also uses two VAD model and two wakeword detection models. These all don't work solely in the browser.
